send-nft/index.js

```diff
@@ -108,36 +104,65 @@ async function run() {
     const signingClient = await createNFTSigningClient(signer);
     const client = signingClient.getSigningStargateClient();
 
+    const hasCsvMemo = !!data.find((e) => !!e.memo);
+
     const msgAnyArray = [];
-    data.forEach((e) => {
+    for (let i = 0; i < data.length; i += 1) {
+      const e = data[i];
       const removed = nftsDataObject[e.classId].splice(0, 1);
-      const msgAny = formatMsgSend(
-        firstAccount.address,
-        e.address,
-        e.classId,
-        removed[0].id,
-      );
-      msgAnyArray.push(msgAny);
-    });
+      if (hasCsvMemo) {
+        const tx = await client.sign(
+          firstAccount.address,
+          msgAnyArray,
+          {
+            amount: [
+              {
+                denom: DENOM,
+                amount: `${new BigNumber(1).shiftedBy(amountNeedDigits).toFixed(0)}`,
+              },
+            ],
+            gas: `${new BigNumber(1).shiftedBy(gasNeedDigits).toFixed(0)}`,
+          },
+          MEMO,
```

Comment:
Should use the memo from csv?
```suggestion
          data[i].memo || MEMO,
```